README.md

```diff
@@ -35,10 +35,11 @@ jobs:
     permissions:
       contents: write
       id-token: write
+      pull-request: write
```

Comment:
For visibility: If I'm not mistaken, this bug is related and the linked comment explains why this is the proper fix:
https://github.com/actions/labeler/issues/12#issuecomment-1095272652

cc @jamesmbourne

Reply:
Fixed the labeler in content-planning-api with this: https://github.com/contentful/content-planning-app/commit/22c78f30698a9f36a044103e8377747bb2f05ae3